lab-nikko/index.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+const greet = require('./lib/greet.js');
+
+if(process.argv[2]) var name = process.argv[2];
+if(process.argv[2] === undefined) var name = 'Carne Asuhdude';
```

Comment:
There is nothing wrong with this.  It works the way it is.  However there is another way to do this variable assignment that is a fairly common practice.  Look up 'ternary operator'.  I'm not sure if you have used it before but it is quite useful.  If you can't find what I am talking about and are still curious let me know and I will show you.